src/lib.rs

```diff
@@ -215,6 +215,9 @@ pub trait PrimeField: Field + From<u64> {
         !self.is_odd()
     }
 
+    /// Returns the field characteristic; the modulus.
+    fn char() -> Self::Repr;
```

Comment:
What use case do you have for this API, that isn't served by `PrimeFieldBits::char_le_bits`?